corehq/apps/userreports/sql/adapter.py

```diff
@@ -211,8 +211,14 @@ def bulk_delete(self, docs, use_shard_col=True):
         table = self.get_table()
         doc_ids = [doc['_id'] for doc in docs]
         delete = table.delete(table.c.doc_id.in_(doc_ids))
-        with self.session_context() as session:
-            session.execute(delete)
+        try:
+            with self.session_context() as session:
+                session.execute(delete)
+        except ProgrammingError as e:
+            if not self.table_exists:
+                return
+            else:
+                raise e
```

Comment:
nit:

Just noting that the else here is redundant. And a comment could be useful.

```
if not self.table_exists:  # ignore error if table is missing
   return
raise e
```